notes.txt

```diff
@@ -0,0 +1 @@
+notes
```

Comment:
**issue**: The file `notes.txt` does not contain any implementation related to the task of adjusting a CV based on a job offer URL. Please ensure that the relevant code is included in the pull request.